### PR TITLE
Fix API schema

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
There were some issues with the API schema and warnings when seen in Stoplight Studio

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: src/specs/api.v2.yaml [hunk 1]
```diff
@@ -774,10 +832,29 @@ paths:
...
+        in: path
+        description: Key of the metadata track to update.
+        required: true
+        schema:
+          type: string
     patch:
       summary: Updates the value that a sample has in a metadata track in particular
       description: Updates a metadata tracks properties
-      operationId: patchMetadataTrack
+      operationId: patchSampleMetadata
```
Comment (cosa65) [inline src/specs/api.v2.yaml]: Nice! by the way, maybe could we make it `patchSampleInMetadataTrackValue`? Or change `metadataTrack#patchSampleInMetadataTrackValue ` to `metadataTrack#patchSampleMetadata`

File: src/specs/api.v2.yaml [hunk 2]
```diff
@@ -969,6 +1061,8 @@ components:
       $ref: './models/experiment-bodies/ExperimentInfo.v2.yaml'
     PatchExperiment:
       $ref: './models/experiment-bodies/PatchExperiment.v2.yaml'
+    GetExperimentUsers:
+      $ref: './models/access-bodies/ExperimentUsers.v2.yaml'
```
Comment (cosa65) [inline src/specs/api.v2.yaml]: Call th file `GetExperimentUsers`?

File: src/specs/api.v2.yaml (hunk 1 above)
Comment (aerlaut) [reply src/specs/api.v2.yaml]: changed

File: src/specs/api.v2.yaml (hunk 2 above)
Comment (aerlaut) [reply src/specs/api.v2.yaml]: This has been remove in a previous change

**Review by StefanBabukov — APPROVED** [review]

**Pull request merged**